addons/point_of_sale/static/src/js/Screens/ProductScreen/ProductScreen.js

```diff
@@ -140,8 +140,6 @@ odoo.define('point_of_sale.ProductScreen', function(require) {
 
                 if(lastId != selectedLine.cid)
                     this._showDecreaseQuantityPopup();
-                else if(currentQuantity === 1 && event.detail.buffer < 1)    // Allow to put a value below 1 for the last product entry.
-                    this._setValue(event.detail.buffer);
                 else if(currentQuantity < event.detail.buffer)
                     this._setValue(event.detail.buffer);
                 else if(event.detail.buffer < currentQuantity)
```

Comment:
Shouldn't we also allow any input if the selected line is the last line? Why do we need to show the popup if the buffer value is less than the current quantity? Shouldn't we just allow 0.9 to be the quantity of the last line without showing the popup?

Reply:
This is just similar to the other comment. I missed the fact that decreasing the quantity is not allowed. But what about the products that are allowed to be fractional (e.g. 0.5, 0.67, etc)? When we click them, they are automatically 1. Does this mean that when we want to modify their quantity after adding, a popup will always appear? An example workflow will be:
1. click a product (line with quantity = 1 is added)
2. press '.', popup will show immediately. I think that this is inconvenient.

perhaps we should allow any modification if it is the last line, regardless of quantity being inputted in the buffer. what do you think?

Reply:
We're not allowed to decrease quantity once it's set.
So even the current behavior is a "kind of trick" because the value is decreased, but we accepted to allow it for the very first input.
So if we can change last line regardless the quantity, it means that we can set it to 40 and then to 20. Which is not allowed.